serverless/pages/what-is-elasticsearch-serverless.asciidoc

```diff
@@ -1,32 +1,61 @@
-
-.Understanding {es-serverless}
-[IMPORTANT]
+[TIP]
 ====
-Refer to <<elasticsearch-differences,Serverless differences>> and <<elasticsearch-technical-preview-limitations,Technical preview limitations>> for important details, including features and limitations specific to {es-serverless}.
+If you haven't used {es} before, you might want to learn the basics in the https://www.elastic.co/guide/en/elasticsearch/reference/current/elasticsearch-intro.html[main {es} documentation].
 ====
 
-[discrete]
-== Get started
+{es} is an open source distributed search and analytics engine, scalable data store, and vector database. It’s optimized for speed and relevance on production-scale workloads. Use {es} to search, index, store, and analyze data of all shapes and sizes.
+
+{es} is one of the three available project types on <<intro,{serverless-full}>>.
 
-* <<elasticsearch-get-started,*Create your {es} project*>>: Create your first {es} project.
-* <<elasticsearch-ingest-your-data,*Ingest your data*>>: Learn how to get your data into {es}.
+When using {es} on {serverless-full} you don’t need to worry about managing the infrastructure that keeps {es} distributed and available: nodes, shards, and replicas. These resources are 100% automated on the serverless platform, which is designed to scale up and down with your workload.
+
+This allows you to focus on building your search applications and solutions, without worrying about the underlying infrastructure.
 
 [discrete]
-== How to
+[[elasticsearch-overview-get-started]]
+== Get started 
+
+[cols="2"]
+|===
+| 🚀
+a| [.card-title]#<<elasticsearch-get-started,*Create project →*>># +
+Get started by creating your first {es} project on serverless.
 
-* <<elasticsearch-search-your-data,*Search your data*>>: Build your queries to perform and combine many types of searches.
-* <<elasticsearch-explore-your-data,*Explore your data*>>: Search, filter your data, and display your findings.
-* <<elasticsearch-explore-your-data-alerting,*Manage alerts*>>: Create rules to detect complex conditions and trigger alerts.
-* <<elasticsearch-dev-tools,*Use developer tools*>>: Send requests with Console and profile queries with Search Profiler.
-* <<elasticsearch-manage-project,*Manage your project*>>: Manage user access, billing, and check performance metrics.
+| 🔌
+a| [.card-title]#<<elasticsearch-get-started,*Connect to {es} →*>># +
+Learn how to connect your applications to your {es-serverless} endpoint.
 
+// TODO add coming link to new page about connecting to your serverless endpoint
+// <<elasticsearch-connecting-to-es-serverless-endpoint,*Connect your application →*>>
+
+| 🔄
+a| [.card-title]#<<elasticsearch-ingest-your-data,*Ingest data →*>># +
+Learn how to get your data into {es} and start building your search application.
+
+| 🛝
+a| [.card-title]#https://www.elastic.co/guide/en/kibana/master/playground.html[*Try Search Playground →*]# +
```

Comment:
is "search" part of the official title? I don't see it anywhere on the linked doc

```suggestion
a| [.card-title]#https://www.elastic.co/guide/en/kibana/master/playground.html[*Try the Search Playground →*]# +
```